torch/nn/quantized/modules/conv.py

```diff
@@ -83,10 +83,15 @@ def __init__(self, in_channels, out_channels, kernel_size, stride=1,
             [out_channels, in_channels // self.groups, self.kernel_size[0],
                 self.kernel_size[1]],
             scale=1, zero_point=0, dtype=torch.qint8)
+        self.weight_scale = 1.0
```

Comment:
May be we can add a check here to make sure that the qscheme of weights is kPerTensorAffine.

Reply:
Currently, perTensorAffine is the only quantization type, so this is not needed. We can add checks for perTensorAffine as we start implementing perChannelQuantization support in Conv2d as a later task.

Reply:
I meant adding an assert to error out explicitly if these modules are called with per channel quantization. Otherwise it might exit with some unrelated error.